### ISSUE TITLE
java.util.NoSuchElementException spammed on console

### ISSUE THREAD

**Issue by ghost** [Eluinhost/Scatterer#1]
```
[11:00:09 WARN]: [Scatterer] Task #197 for Scatterer v1.1.1 generated an exception
java.util.NoSuchElementException
        at java.util.LinkedList.removeFirst(Unknown Source) ~[?:1.8.0_73]
        at java.util.LinkedList.pop(Unknown Source) ~[?:1.8.0_73]
        at gg.uhc.scatterer.teleportation.Teleporter$TeleportTask.run(Teleporter.java:85) ~[?:?]
        at org.bukkit.craftbukkit.v1_9_R2.scheduler.CraftTask.run(CraftTask.java:58) ~[patched_1.9.4.jar:git-Paper-756]
        at org.bukkit.craftbukkit.v1_9_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:352) [patched_1.9.4.jar:git-Paper-756]
        at net.minecraft.server.v1_9_R2.MinecraftServer.D(MinecraftServer.java:795) [patched_1.9.4.jar:git-Paper-756]
        at net.minecraft.server.v1_9_R2.DedicatedServer.D(DedicatedServer.java:403) [patched_1.9.4.jar:git-Paper-756]
        at net.minecraft.server.v1_9_R2.MinecraftServer.C(MinecraftServer.java:723) [patched_1.9.4.jar:git-Paper-756]
        at net.minecraft.server.v1_9_R2.MinecraftServer.run(MinecraftServer.java:622) [patched_1.9.4.jar:git-Paper-756]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_73]
```

Latest commit, Paper 1.9.4

**Comment by Eluinhost** [comment]
This seems to happen when you try scattering without any targetted players and noone is in the world, I've added a check that gives an error when noone is targetted instead of trying to scatter anyway

**Comment by Eluinhost** [comment]
Released as 1.1.2 https://github.com/Eluinhost/Scatterer/releases/tag/1.1.2
